### ISSUE TITLE
tfctl to generate terarform files

### ISSUE THREAD

**Issue by k0da** [weaveworks/tf-controller#535]
During execution several files are generated: 
* values.json
* backend.json

It would be really helpful to be able to generate files for local execution/inspection

**Comment by nogweii** [comment]
Indeed, I'm considering using tf-controller but this is a personal blocker.

There are a bunch of helpful tools that wrap calling the binary to help manage or debug state, and without these files the local execution of `terraform` would fail to work. I understand that this is at risk for users to break things, but I think that'd be safe enough with appropriate warnings and documentation.

**Comment by chanwit** [comment]
yep, I agreed.
we're planning to support this kind of code generation and inspection.

in the mean time, we have a feature called `break-the-glass` to help you troubleshoot your problems.
